.travis/deploy_npm

```diff
@@ -36,22 +36,56 @@ fi
 exe node -v
 exe npm  -v
 
+echo ">>> Installing dependencies..."
 exe npm install
-exe npm install --no-save @qooxdoo/compiler
-
-exe npx qx --version
-exe npx qx pkg install -v 
-#compile everything
-exe npx qx compile --config-file=compileServer.json --target=build 
-#exe npx qx compile --config-file=compile.json       --target=build 
-#remove unnecessary stuff
-#exe find ./apps/transpiled -type f -name '*.map' -delete
-#exe find ./apps/transpiled -type f -name '*.js'  -delete
-#copy source
+
+echo ">>> Installing github.com/qooxdoo/qooxdoo-compiler"
+exe rm -rf ./qooxdoo-compiler
+exe git clone https://github.com/qooxdoo/qooxdoo-compiler.git --depth=1 --single-branch ./qooxdoo-compiler
+exe cd ./qooxdoo-compiler
+exe npm install
+QX=`pwd`/qx
+exe cd ..
+exe $QX --version
+
+echo ">>> Compiling & deploying server library with the compiler master using current qx_server library from NPM..."
+exe $QX compile --config-file=compile-server.json --target=build
+exe mkdir -p ./lib
+exe rm -rf ./lib/*
+exe cp -R compiled/build/qx_server ./lib
+
+echo ">>> Running Compiler Tests"
+exe cd ./qooxdoo-compiler
+exe test/test.linux.sh
+exe cd ..
+
+echo ">>> Compiling & deploying server library with compiler master using the newly built qx_server library..."
+exe rm -rf ./qooxdoo-compiler/node_modules/@qooxdoo/framework/lib
+exe cp -fR lib ./qooxdoo-compiler/node_modules/@qooxdoo/framework
+exe cp -f package.json Manifest.json ./qooxdoo-compiler/node_modules/@qooxdoo/framework
+exe rm -rf compiled
+exe $QX compile --config-file=compile-server.json --target=build
+exe $QX deploy --config-file=compile-server.json
```

Comment:
IMHO here is --out=?? missing

Reply:
No, that's in the `compile.json`'s `deployPath` directive. That's the beauty of it and that's why they also need to be part of this PR.